test-with-jest/src/mock-function/index.ts

```diff
@@ -0,0 +1,17 @@
+import { connectToContainer } from './data/connect-to-cosmos';
+import { createTestInput } from './data/fake-data';
+import { DbDocument, DbError, VerificationErrors } from './data/model';
+import { insertDocument } from './lib/insert';
+
+async function main(): Promise<DbDocument | DbError | VerificationErrors> {
+  const container = await connectToContainer();
+  const input = createTestInput();
+  return await insertDocument(container, input);
+}
+
+main()
```

Comment:
nit: could be a top level await with try/catch for consistency, since you're targetting Node 22